templates/policies/privacy.hbs

```diff
@@ -0,0 +1,203 @@
+          using a GitHub account, we receive your GitHub username and avatar. If
+          you share a display name or public email address in your GitHub public
+          profile, we also receive that information.
+        </p>
+      </dd>
+
+      <dt>Email addresses:</dt>
+      <dd>
+        <p>
+          You must have a verified email address to publish a crate. We receive
+          any public email address associated with your GitHub account. You can
+          also choose to submit a different address to associate with your
+          Crates.io activity. We will only use your email address to contact you
+          about your account.
+        </p>
+      </dd>
+
+      <dt>Visitor logs:</dt>
+      <dd>
+        <p>
+          When you visit Crates.io, we receive your IP address and user-agent
+          header as part of our standard server logs. We store these logs for 1
+          year.
+        </p>
+      </dd>
+
+      <dt>Information uploaded to crates:</dt>
+      <dd>
+        <p>
+          All crates on Crates.io are public, including the list of crate owners’
+          user names and its upload date. Anyone may view or download a crates’
+          contents. Because of the public nature of Crates.io any personal data
+          you might include in a Cargo.toml file uploaded to a create will be
+          publicly available. For example, if an email address is in the author’s
+          field in the Cargo.toml file, that email address will also be public.
+        </p>
+        <p>
+          Due to its public nature, be aware if you include any private
+          information in a crate that information may be indexed by search
+          engines or used by third parties. Sensitive information should not be
+          included in a crate file.
+        </p>
+      </dd>
+
+      <dt>Publication of site-related data: </dt>
+      <dd>
+        <p>
+          Crates.io uses the Google Visualization API to create graphs for each
+          crate showing downloads over the last 90 days. Those graphs can be seen
+          at the bottom of each crate’s page.
+        </p>
+      </dd>
+    </dl>
+
+    <header>
+      <h2 id="docs-rs">docs.rs</h2>
+      <div class="highlight"></div>
+    </header>
+
+    <p>
+      <a href="https://docs.rs">Docs.rs</a> is managed by the members of the
+      <a href="{{baseurl}}/governance/teams/dev-tools#rustdoc">Rustdoc team</a>
+      and <a href="{{baseurl}}/governance/teams/core">Rust core team</a>. The
+      project collects and uses data as described in this privacy notice.
+    </p>
+
+    <p>
+      Docs.rs is an open source project to host documentation of crates for the
+      Rust Programming Language. It automatically builds crates' documentation
+      released on <a href="https://crates.io">crates.io</a> using the nightly
+      release of the Rust compiler. All information from crates that is
+      published on docs.rs is also available publicly on crates.io.
```

Comment:
I think this isn't strictly true as docs.rs also scrapes GitHub and such for repository information (e.g., github star count). But docs.rs collects that information from public APIs (there's no accounts or such).